lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -650,7 +650,18 @@ def upload_students_csv(_xmodule_instance_args, _entry_id, course_id, task_input
     task_progress.update_task_state(extra_meta=current_step)
 
     # Perform the upload
-    upload_csv_to_report_store(rows, 'student_profile_info', course_id, start_date)
+    # Note: values in each row are expected to be unicode, str, int, or
```

Comment:
The first list (header row) contains strings representing attributes on a student. It is currently `['id', 'username', 'name', 'email', 'language', 'location', 'year_of_birth', 'gender', 'level_of_education', 'mailing_address', 'goals']`, so no unicode.

The actual value of these features are gotten from `getattr(student, feature)` (see `enrolled_students_features`), and make up the rest of the rows. I don't see how those could be encoded in anything but ascii (famous last words?).